reddit_liveupdate/public/static/css/liveupdate-public.less

```diff
@@ -0,0 +1,123 @@
+// CSS in here is imported into reddit's main CSS file, for inclusion of live features into reddit proper
```

Comment:
This is a tiny bit janky. I wanted to reuse the statusbar class for the homepage (since it uses the same style), but reddit live's CSS is (reasonably) not included in the default includes. This pulls this file out and includes it in both. That means that the rendered CSS on live pages will duplicate these rules, but that will have no practical effect except a few wasted bits.